.devcontainer/Dockerfile

```diff
@@ -6,6 +6,7 @@ LABEL description="Core development container"
 RUN apt-get update && apt-get -y upgrade
 RUN apt-get install -y ca-certificates curl gnupg lsb-release software-properties-common
 RUN apt-get install -y g++ clang lcov clang-format clang-tidy clangd
+RUN apt-get install -y g++-12-aarch64-linux-gnu qemu-user
```

Comment:
Any reason to add this as a separate layer rather than roll into the above command?

Reply:
No real reason, but thinking about it, the above line can be easily included in an (eventual) arm64 version of the devcontainer whereas this line is specific to this platform. 🤔